test/DynamoCoreWpfTests/NodeAutoCompleteSearchTests.cs

```diff
@@ -349,7 +349,7 @@ public void SearchNodeAutocompletionSuggestions()
 
             // Filter the node elements using the search field.
             searchViewModel.SearchAutoCompleteCandidates("ar");
-            Assert.AreEqual(2 , searchViewModel.FilteredResults.Count());
+            Assert.AreEqual(4 , searchViewModel.FilteredResults.Count());
```

Comment:
@zeusongit Looks like the number diff is expected due to?

Reply:
Because the search term `ar` now returns more than 2 results as the search scope has been increased to cover description and keywords.